lib-extra/src/main/java/com/diffplug/spotless/extra/GitAttributesLineEndings.java

```diff
@@ -56,7 +56,7 @@
 
 /**
  * Uses [.gitattributes](https://git-scm.com/docs/gitattributes) to determine
```

Comment:
I wonder if this markdown link should be an HTML one like `<a href="https://git-scm.com/docs/gitattributes">.gitattributes</a>`, with ".gitattributes" optionally wrapped in a `{@code ... }` block. What do you think?

Reply:
Thanks, good catch! I just fixed this with a regex in https://github.com/diffplug/spotless/pull/917.